news/feature-3703.md

```diff
@@ -0,0 +1,3 @@
+created MQTT destination feature
+
+can be create mqtt destination, send message from syslog-ng to mqtt topic
```

Comment:
Please attach an example configuration snippet, you can use markdown code blocks in this file.

You can check our previous feature descriptions as an example:
https://github.com/syslog-ng/syslog-ng/releases

Reply:
solved by this comment: ea255af65b0ecc8d96932a480e7d13abd1572305

Reply:
This still has some grammatical errors, but I'll just rephrase it in the final NEWS file, if you allow it.